doc/source/whatsnew/v0.20.0.txt

```diff
@@ -308,6 +308,9 @@ Other enhancements
 - ``pd.types.concat.union_categoricals`` gained the ``ignore_ordered`` argument to allow ignoring the ordered attribute of unioned categoricals (:issue:`13410`). See the :ref:`categorical union docs <categorical.union>` for more information.
 - ``pandas.io.json.json_normalize()`` with an empty ``list`` will return an empty ``DataFrame`` (:issue:`15534`)
 - ``pd.DataFrame.to_latex`` and ``pd.DataFrame.to_string`` now allow optional header aliases. (:issue:`15536`)
+- Reshape operations now preserve subclass family. This includes ``DataFrame``
```

Comment:
reshape operations on Series/DataFrame will now preserve subclasses. The following operations ......